### PR TITLE
feat: display when a self client was last active

### PR DESCRIPTION
**Pull request by typfel**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [ ] contains a reference JIRA issue number like `SQPIT-764`
    - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

When available display information about when a client was last active.

### Dependencies

Needs releases with:

- [x] https://github.com/wireapp/kalium/pull/1886

### Attachments

Screenshot_20230718_142534

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [472](https://10.10.124.134/job/AR-build-pipeline/472/) **failed**.

**Review by vitorhugods — APPROVED** [review]
Diving deeper into Android development, aren't you?

Nicely done :D 

-----

The failing `continuous-integration/jenkins/pr-head` check is just because of the git conflict, btw. No worries.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -879,6 +880,9 @@
     <string name="label_code_commit_id">Commit Hash: %1$s</string>
     <string name="label_client_id">Client ID: %1$s</string>
     <string name="label_client_added_time">ADDED</string>
+    <string name="label_client_last_active_label">LAST_ACTIVE</string>
```
Comment (MohamadJaara) [inline app/src/main/res/values/strings.xml]: Is it intentional to have the label in SCREAMING_SNAKE_CASE?

Comment (typfel) [reply app/src/main/res/values/strings.xml]: Oops,no 😬

**Comment by AndroidBob** [comment]
Build [530](https://10.10.124.134/job/AR-build-pipeline/530/) **failed**.

**Review by alexandreferris — APPROVED** [review]

**Review by MohamadJaara — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [540](https://10.10.124.134/job/AR-build-pipeline/540/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.3.0(7081204)(7081204)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.3.0(7081204)(7081204)-staging-compat.apk)

**Pull request merged**